pkg/identifier/expression.go

```diff
@@ -0,0 +1,550 @@
+/*
+Copyright © 2024 Acronis International GmbH.
+
+Released under MIT license.
+*/
+
+package identifier
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/google/uuid"
+)
+
+// Vendor is an application vendor name.
+type Vendor string
+
+// IsWildCard returns true if the vendor name is a wildcard.
+func (v Vendor) IsWildCard() bool {
+	return string(v) == string(Wildcard)
+}
+
+// Application is an application name.
+type Application string
```

Comment:
```suggestion
type Package string
```
Let's rename everywhere to Package